src/reporting/clients/IAggregationsClient.ts

```diff
@@ -0,0 +1,278 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import { AggregationProperty, AggregationPropertyCreate, AggregationPropertyUpdate, AggregationTable, AggregationTableCreate, AggregationTableSet, AggregationTableSetCreate, AggregationTableSetUpdate, AggregationTableUpdate } from "../interfaces/AggregationProperties";
+import type { EntityListIterator } from "../../common/iterators/EntityListIterator";
+
+export interface IAggregationsClient {
+  /**
+     * Gets all AggregationProperties in a given aggregation table. This method returns the full list of Aggregation Properties.
```

Comment:
Indentation of documentation comments looks inconsistent compared to other files.